### DISCUSSION TITLE
How to add background color to a label?

### DISCUSSION THREAD

**Discussion by lassepe** [JuliaPlots/Makie.jl#2564] [Q&A]
I would like to add background color to a label to highlight it more clearly from the background. However, `Makie.Label` does not seem to support this (?). A work-around that kind of works is to manually render `Makie.Box` at the same position but `Makie.Box` does not seem to respond to any settings of the `padding` kwarg so I find it difficult to format it properly. Is there a better way to do this?

**Comment by kbarros** [comment]
I also found myself needing this. More recent discussion with examples appears here: https://discourse.julialang.org/t/fill-a-bounding-box-behind-text-in-makie/101502/2
